types/color-name/index.d.ts

```diff
@@ -0,0 +1,157 @@
+// Type definitions for color-name 1.1
+// Project: https://www.npmjs.com/package/color-name
+// Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Red = number;
```

Comment:
@andy-ms I fix it with `RGB`, but I think specifying `Red`, `Green`, and `Blue` is more informative.